web/static/js/phoenix_html.js

```diff
@@ -1,13 +1,20 @@
-function didHandleSubmitLinkClick(element){
+function isLinkToSubmitParent(element) {
+  var isLinkTag = element.tagName === 'A';
+  var shouldSubmitParent = element.getAttribute('data-submit') === 'parent';
+
+  return isLinkTag && shouldSubmitParent;
+}
+
+function didHandleSubmitLinkClick(element) {
   while(element) {
-    if(element.matches && element.matches('a[data-submit=parent]')){
+    if(isLinkToSubmitParent(element)) {
       var message = element.getAttribute('data-confirm');
       if (message === null || confirm(message)) {
-        element.parentNode.submit();
+        element.parentElement.submit();
```

Comment:
@jsteiner the problem seems to be IE as usual, see second response: http://stackoverflow.com/questions/8685739/difference-between-dom-parentnode-and-parentelement :(